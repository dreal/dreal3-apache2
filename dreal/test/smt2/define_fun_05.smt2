(set-logic QF_NRA)
(define-fun x () Real 5.0)
(assert (= x 5.0))  ; sat
(check-sat)
(get-model)
