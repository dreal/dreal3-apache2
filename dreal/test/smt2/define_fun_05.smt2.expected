delta-sat with delta = 0.001
(model
  (define-fun x () Real 5)
)
